### PR TITLE
Decouple Open Y Activity Finder module to separate repository.

### PR DESCRIPTION
**Pull request by duozersk**
Action on https://docs.google.com/document/d/1W5YeRtzv0uRtlF0547964eMq1VL2z5aFgLM_EfYQfEg/edit

Released 3.0.0 - https://github.com/ymcatwincities/openy_activity_finder/releases/tag/3.0.0
Removed openy_activity_finder module from Open Y profile
Added new dependency - "ymcatwincities/openy_activity_finder": "^3.0"

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: composer.json [hunk 1]
```diff
@@ -103,7 +103,8 @@
         "npm-asset/jquery.easing": "^1.4",
         "npm-asset/slick-carousel": "^1.8",
         "oomphinc/composer-installers-extender": "^1.1 || ~2.0.0",
-        "symfony/dom-crawler": "~2.8 || ~3.0 || ~4.2"
+        "symfony/dom-crawler": "~2.8 || ~3.0 || ~4.2",
+        "ymcatwincities/openy_activity_finder": "^3.0"
```
Comment (podarok) [inline composer.json]: some day we'll have "^3.0 || ^4.0" and then this line probably should go ))))

**Review by podarok — DISMISSED** [review]
code wise - looks good

**Comment by podarok** [comment]
ok to test

File: composer.json (hunk 1 above)
Comment (duozersk) [reply composer.json]: Right you are ;)

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2791/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2791/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2790/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2790/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2790/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2791/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2790/

File: composer.json [hunk 2]
```diff
@@ -104,7 +104,7 @@
         "npm-asset/slick-carousel": "^1.8",
         "oomphinc/composer-installers-extender": "^1.1 || ~2.0.0",
         "symfony/dom-crawler": "~2.8 || ~3.0 || ~4.2",
-        "ymcatwincities/openy_activity_finder": "^3.0"
+        "ymcatwincities/openy_activity_finder": "^3.0 || ^4.0"
```
Comment (podarok) [inline composer.json:107]: too early . Let's have 4.0, let's discuss our release plan and than add it. Sorry for confusion

**Review by podarok — CHANGES_REQUESTED** [review]

**Pull request merged**